docs/documentation/maya/simshape.md

```diff
@@ -1,16 +1,16 @@
-# How To Use
+# Introduction to Simshape in Maya
 
 **Simshape** is a Maya deformer to get facial simulation in the animation rig of an asset. Given a facial expression, the deformer is able to compute the activation of the vertices in order to emulate the changes in the rigidity of the skin. As result, the dynamics of the simulated skin mimic the behaviour of internal muscles contracting.
 
 During the simulation, the solver reduces the inertias of the vertices with higher values of activation, while it computes standard simulation in the vertices that are not activated. One of the key features of Simshape is the ability to extract muscle information directly from the neutral geometry and the set of deformed geometries with the facial expressions using Machine Learning techniques.
 
 ## Requirements
```

Comment:
I am seeing that Collider and Rest Collider are explained in detail below but don't you think we should also add them to this list of requirements? If we finally add it I think it would be good to mention that these two meshes must have the same number of vertices but it can be different from the number of vertices of the other 4 inputs.

Reply:
Please, check that these comments have been addressed and take note if we have to add similar comments for other deformers/doc pages